skimage/exposure/_adapthist.py

```diff
@@ -38,10 +39,14 @@ def equalize_adapthist(image, ntiles_x=8, ntiles_y=8, clip_limit=0.01,
     ----------
     image : array-like
         Input image.
-    ntiles_x : int, optional
-        Number of tile regions in the X direction.  Ranges between 1 and 16.
-    ntiles_y : int, optional
-        Number of tile regions in the Y direction.  Ranges between 1 and 16.
+    kernel_size: integer or 2-tuple
+        Defines the shape of contextual regions used in the algorithm.
+        If an integer is given, the shape will be a square of
+        sidelength given by this value.
```

Comment:
@blink1073 I think this should be at the end of the argument list, but I understand what you're doing. If you think this is better, I'll allow it. =)